terraform-aws-github-runner/modules/runners-instances/templates/user-data.sh

```diff
@@ -138,6 +138,11 @@ fi
 
 ${post_install}
 
+sudo fallocate -l 3G /swapfile
```

Comment:
Should we have a swap file that scales with the amount of memory on the runner, i.e. https://github.com/pytorch/pytorch/pull/142293/files#diff-b317d4da565a9e329ccf67e669c2ff1f4d4bc5fb0ffa4d74132545ad66f84339R229?